### DISCUSSION TITLE
Typography: option to suppress color styles

### DISCUSSION THREAD

**Discussion by MichaelAllenWarner** [tailwindlabs/tailwindcss#5711] [Ideas]
As mentioned and discussed at some length in [this issue](https://github.com/tailwindlabs/tailwindcss-typography/issues/102), it's rather difficult to customize the color-scheme in Tailwind Typography. I left a [comment](https://github.com/tailwindlabs/tailwindcss-typography/issues/102#issuecomment-932259011) there with an idea for making it easier, and I thought I'd duplicate it here as a feature request:

Yes, customizing the color-scheme is a pain.

I propose an option to simply _suppress_ the default `color` rules entirely.

Enabling such an option would mean sacrificing different shades for bullets and the like, but as nice as the default color scheme is, I'd _gladly_ trade those subtleties for the ability to do something like `<div class="prose prose-white">`, with a config that could be as simple as:

```js
typography: {
  white: {
    css: {
      color: 'white',
      'a:hover': {
        color: 'whatever'
      }
    }
  }
}
```

**Comment by istr** [comment]
I would go one step further and make this behavior (leave the colors alone) the default behavior for tailwindcss typography.

From my perspective, typography is primarily about optimizing the spatial organization of text. Common definitions of typography and a look at the history of typography confirm this view.

The question of color is resolved in the context of branding or theming. Typography and coloring must be coordinated, but they should not directly influence each other. The only direct connection between typography and coloring concerns the contrast between font and background, which has a massive impact on readability.

There are plugins for talewindcss that can be used to implement theming. Interoperability with tailwind typography suffers especially from the fact that the typography plugin specifies a "default" color scheme that does not play well with the logic of these plugins (for example [daisyUI](https://daisyui.com/)).

**Comment by radiosilence** [comment]
Is there an easy way to do this yet?
